resources/lang/de/dashboard.php

```diff
@@ -118,7 +118,7 @@
     // Subscribers
     'subscribers' => [
         'subscribers'  => 'Abonnenten',
-        'description'  => 'Abonnenten erhalten e-Mail-Updates, wenn Vorfälle erstellt werden.',
+        'description'  => 'Abonnenten erhalten E-Mail Updates wenn Vorfälle erstellt werden.',
```

Comment:
One question here for consistency, as I have stumbled over this several times. How do we spell "E-Mail"? My preference would be indeed **"E-Mail"**, but the translator before me used **"eMail"** in Transifex, so I continued with that. But this line now changes that. So which one do you guys prefer:

  * **E-Mail**
  * **Email**
  * **eMail**
  * **e-Mail**
  * **email**

... Maybe some possible spelling option I forgot?